client/src/utils/handleUpdateTasks.js

```diff
@@ -75,7 +70,35 @@ export const addDashboard = (title, cb) => {
       cb(res.result);
     })
     .catch(err => {
-      console.log(err);
+      handleError(err);
+    });
+};
+
+export const getDashboard = (dashboardId, cb) => {
+  let body = {
+    dashboardId
+  };
+
```

Comment:
why are you making a post request to get dashboard? also dashboard Id is in params so no need to add it to the body variable you have. I would change this so you have a route to get dashboard on the backend as well as a route to post to dashboard

Reply:
thanks. fixed.

Reply:
In terms of drawer. I'll create other branch